lib/v2/bilibili/cache.js

```diff
@@ -8,6 +8,30 @@ module.exports = {
             return response.headers['set-cookie'];
         });
     },
+    getStaticVerifyString: (ctx) => {
+        const key = 'bili-verify-string';
+        return ctx.cache.tryGet(key, async () => {
+            const cookie = await module.exports.getCookie(ctx);
+            await got(`https://space.bilibili.com/1/`, {
+                headers: {
+                    Referer: 'https://www.bilibili.com/',
+                    Cookie: cookie,
+                },
+            });
+            const response = await got(`https://s1.hdslb.com/bfs/static/jinkela/space/9.space.ecceb61b045ad88627ce742a4fdd10a68c0ab747.js`, {
+                headers: {
+                    Referer: `https://space.bilibili.com/1/`,
+                },
+            });
+            const array = JSON.parse(response.body.match(/\[(?:\d+,){63}\d+\]/));
+            const r = "162d9a373be342dbb91eff71ed0a894cf53968b083f2431bb0efed8606707956";
```

Comment:
This value is not fixed. The value is generated from the concatenation of filename of 2 non-exist items stored in the browsers' local storage.

Line 11593 of `9.space.ecceb61b045ad88627ce742a4fdd10a68c0ab747.js`
```js
                key: "_setLocalWbi",
                value: function(t) {
                    var e, n;
                    null != t && t.wbi_img && (window.localStorage.setItem("wbi_img_url", null === (e = t.wbi_img) || void 0 === e ? void 0 : e.img_url),
                    window.localStorage.setItem("wbi_sub_url", null === (n = t.wbi_img) || void 0 === n ? void 0 : n.sub_url))
                }
```
![image](https://user-images.githubusercontent.com/11386903/208987894-76e6ddee-aaaa-45be-a536-9daf3161a316.png)
---
Both URLs are the response of Line 11622 of `9.space.ecceb61b045ad88627ce742a4fdd10a68c0ab747.js`
```js
                key: "_getNav",
                value: function() {
                    var t = this;
                    this.cache ? this._notice(this.cache) : this.locked || (this.locked = !0,
                    f({
                        method: "GET",
                        url: m
                    }).then((function(e) {
                        t.cache = e,
                        t.isLogin = e.data && e.data.isLogin,
                        t._notice(e, !0),
                        t._setLocalWbi(e.data)
                    }
                    )).catch((function(e) {
                        t._notice({
                            code: 500,
                            message: "接口异常"
                        }, !0)
                    }
                    )))
                }
```
`m = '//api.bilibili.com/x/web-interface/nav'`